app/models/bus_stops_route.rb

```diff
@@ -9,4 +9,45 @@ class BusStopsRoute < ApplicationRecord
   validates :direction, presence: true
   validates_uniqueness_of :sequence, scope: %i[route direction]
   validates_uniqueness_of :bus_stop, scope: %i[route direction]
+
+  def self.import(stop_hash)
```

Comment:
It might be nice to outline the algorithm in a block comment before the method body.